src/scene/camera.js

```diff
@@ -51,6 +51,10 @@ class Camera {
         this._renderTarget = null;
         this._scissorRect = new Vec4(0, 0, 1, 1);
         this._scissorRectClear = false; // by default rect is used when clearing. this allows scissorRect to be used when clearing.
+        this._usePhysicallyBasedCamera = false;
```

Comment:
I wonder if this should be exposed in the LightingParameters perhaps (which is part of the Scene). If this is per Camera, what happens if one camera is physical and one not, would they be able to use the lights, where some are physical and some not? Perhaps we need this per scene setting that would apply to both lights and cameras?

Reply:
Ah yeah, good point. Let's move it out to scene settings instead!